docs/source/tutorials/parsl-rp.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>Instead of using a comment inside the cell, I would use a text cell in the notebook as done above. I would do the same for the following code cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ExaWorks/SDK/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>